### ISSUE TITLE
Deprecated warning w/ node 6

### ISSUE THREAD

**Issue by black-snow** [chesstrian/JSCryptor#16]
> (node:22968) DeprecationWarning: crypto.pbkdf2 without specifying a digest is deprecated. Please specify a digest

```
node -v
v6.2.2
```

Guess we should pass `SHA-1` explicitly.

See https://github.com/ericelliott/credential/issues/70 as a reference.

**Comment by black-snow** [comment]
This indeed fixes it, thanks for the ultra-fast response ;)

**Comment by amnotafraid** [comment]
When I changed over from Node 4.2 to Node 6.0, I started getting this warning. From the comments, it was unclear which version or even which package I needed to update to in order to not get the warning.

However, I changed from this:
```
  1 this.password = crypto.pbkdf2Sync (
  2                   password,
  3                   this.salt,
  4                   1000,
  5                   64).toString('hex');
```
to this:
```
  1 this.password = crypto.pbkdf2Sync 
  2       (password,
  3        this.local.salt,
  4        1000,
  5        64,
  6        'sha1').toString('hex');
```
and I was okay.

**Comment by chesstrian** [comment]
@amnotafraid Your fix is totally right and it's already included in the library on version 0.0.12, feel free to check and reopen the issue if it's still happening.

Regards.
